### ISSUE TITLE
Appearance of 'Downloaded only' and 'Incognito mode' toggle is broken

### ISSUE THREAD

**Issue by AgrMohit** [jobobby04/TachiyomiSY#589]
### Steps to reproduce

Go to *More* section in bottom bar. On the toggle for *Downloaded only* and *Incognito mode*, *true* and *false* are written on top of the toggles. Furthermore on tapping the toggles, the colours change and they work as expected but the position of the toggles remains the same until I switch to some other page and come back in which case the position becomes as expected.

### Expected behavior

*true* and *false* should not be written on the toggles and they should move when the state is changed.

### Actual behavior

*true* and *false* is written on top of toggles and they dont move when state is changed.

### Tachiyomi version

r451

### Android version

Android 12

### Device

Realme 7 Pro

### Other details

![Screenshot_2022-04-24-23-49-26-93_a603e4da04bada03cae438f177815547](https://user-images.githubusercontent.com/32242258/164990722-41f07316-9747-44e6-ae9b-0f09458c5429.jpg)
![Screenshot_2022-04-24-23-49-40-95_a603e4da04bada03cae438f177815547](https://user-images.githubusercontent.com/32242258/164990725-cdb82553-dbb0-4198-aad7-48a64ef8f266.jpg)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[1.8.3](https://github.com/jobobby04/tachiyomisy/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by jobobby04** [comment]
Known main Tachi bug, we are waiting on a fix by google.

**Comment by jobobby04** [comment]
For reference 
![rB27OPp 1](https://user-images.githubusercontent.com/17078382/164990845-8f6edd45-ef06-466a-8370-c2cedc9b2f02.png)
